start.js

```diff
@@ -1,53 +1,55 @@
 #!/usr/bin/env node
-const http = require('https');
-const fs = require('fs');
-const path = require('path');
-const { spawn } = require('child_process');
+const http = require("https");
+const fs = require("fs");
+const path = require("path");
+const { spawn } = require("child_process");
 
-const decompress = require('decompress');
-const decompressTargz = require('decompress-targz');
+const decompress = require("decompress");
+const decompressTargz = require("decompress-targz");
 
 const args = process.argv.slice(2);
-const version = args[0] || '11.0.0';
+const version = args[0] || "12.0.1";
 
-const folder = 'server';
-const fileName = path.join(folder, 'keycloak.tar.gz');
+const folder = "server";
+const fileName = path.join(folder, "keycloak.tar.gz");
```

Comment:
Let's change this line so that we always download the right version and don't skip the download step because the old file is there
```suggestion
const fileName = path.join(folder, `keycloak-${version}.tar.gz`);
```